content/reference/user/organization/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Organization"
+description: "Learn how to access organization details."
+date: 2021-08-04T14:29:04-00:00
+lastmod: 2021-08-04T14:29:04-00:00
+weight: 10081
+---
+
+To consult information regarding an organization, click on the **Organization name link**.
```

Comment:
To view information about an organization

Reply:
Done